### DISCUSSION TITLE
Database functions definition code editor

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#16839] [Contribute to Supabase]
The editor area has an unusually short, fixed height. 

We should look at options here for making that area scrollable or expandable, or at the very least, somewhat taller.

![CleanShot 2023-08-11 at 09 40 34@2x](https://github.com/supabase/supabase/assets/105593/e810f9af-d29f-4503-a5bf-25906aa9c7c4)

**Comment by imSuMN** [comment]
hey i am new to open source. Can i work on this issue?

**Comment by imSuMN** [comment]
The height of the window is set fixed  (h-40), we can use ( h-60 resize-y).

**Comment by Gingaaa** [comment]
Hey I have worked on around 7-8 company based project, but never countribute to open source, this is my first time can you assign me this task.

**Comment by vibhaw1904** [comment]
Hello @saltcod ,

I'm excited to contribute! Could you kindly assign me the problem related to  database function height resize issue. I believe this aligns well with my skills and interests. Looking forward to making a positive impact!

Thank you,
vibhawraj1904 Hello @saltcod ,

**Comment by Vidit-Goel2k** [comment]
Hello maintainers,

I've been following the discussion on issue #16444, and it looks like the problem described here has been addressed by the changes introduced in PR #16486. The proposed solution in the pull request seems to effectively resolve the issue at hand.

I kindly request the maintainers to review the PR and, if it indeed resolves the problem, consider closing this issue. Your efforts are greatly appreciated!

Thank you,
[Vidit-Goel2k](https://github.com/Vidit-Goel2k)
